### ISSUE TITLE
Wrong loss function

### ISSUE THREAD

**Issue by HC-2016** [moabitcoin/holy-edge#15]
In the original paper:
beta = |Y_| / |Y|
"|Y_| and |Y+| denote the edge and non-edge ground truth label sets, respectively"

This definition is really counter-intuitive to me.
________________________________________________________
In "sigmoid_cross_entropy_balanced":
```
    y = tf.cast(label, tf.float32)

    count_neg = tf.reduce_sum(1. - y)
    count_pos = tf.reduce_sum(y)

    # Equation [2]
    beta = count_neg / (count_neg + count_pos)

    # Equation [2] divide by 1 - beta
    pos_weight = beta / (1 - beta)
```
________________________________________________________
It seems that "sigmoid_cross_entropy_balanced" function in "losses.py" is wrong.
